.circleci/config.yml

```diff
@@ -239,6 +239,19 @@ commands:
              if [ ! -f <<parameters.test_list_file>> ]; then exit 1; fi
              pytest --junitxml=test-results/junit.xml --verbose --timeout 60 --cov-report=xml --cov=./ `cat <<parameters.test_list_file>>`
 
+   setup_pyenv:
+     parameters:
+       version:
+         type: string
+     steps:
+       - run:
+           name: Setup pyenv
```

Comment:
can you add comment on this?

- what does it do?
- what is the parameter
- what files are being saved/changed on the CI machine; are these files/state included in the cache, etc.?